examples/inference/hf_generate.py

```diff
@@ -99,6 +101,7 @@ def run_all(pp_ranks, args):
     parser.add_argument('--chunks', type=int, default=1)
     parser.add_argument('--cuda', type=int, default=int(torch.cuda.is_available()))
     parser.add_argument('--pp_group_size', type=int, default=int(os.getenv("WORLD_SIZE", 4)))
+    parser.add_argument('--index_filename', type=str, default=None, help="The director or url of model's index.json file")
```

Comment:
@jiqing-feng can. you pls add an example of " url of model's index.json file" or we might need to have a script to download the model checkpoint that let us run this e2e.

Reply:
Sorry, it is a mistake and I have changed it. The model cannot be loaded by the URL since all weights are saved in the model.bin, so we need to download the model anyway. Thanks for your reminder.